index.js

```diff
@@ -34,6 +37,11 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+let firstName = "Maya";
+let lastName = "Worth";
+let fullName = firstName + " " + lastName;
+console.log (fullName)
```

Comment:
looks good!
another way you could write this with string literals would be
let fullName = `${firstName} ${lastName`;